### PR TITLE
Add Surya model

### PR DESCRIPTION
**Pull request by as-suvorov**
Ticket: CVS-131138

### CODE REVIEW

File: .ci/ignore_pip_conflicts.txt [hunk 1]
```diff
@@ -14,4 +14,5 @@ notebooks/257-llava-multimodal-chatbot/257-llava-multimodal-chatbot.ipynb # tran
 notebooks/257-llava-multimodal-chatbot/257-videollava-multimodal-chatbot.ipynb # transformers<4.35
 notebooks/273-stable-zephyr-3b-chatbot/273-stable-zephyr-3b-chatbot.ipynb # install requirements.txt after clone repo
 notebooks/279-mobilevlm-language-assistant/279-mobilevlm-language-assistant.ipynb # transformers<4.35
-notebooks/280-depth-anything/280-depth-anything.ipynb # install requirements.txt after clone repo
\ No newline at end of file
+notebooks/280-depth-anything/280-depth-anything.ipynb # install requirements.txt after clone repo
+notebooks/285-surya-line-level-text-detection/285-surya-line-level-text-detection.ipynb # requires python >=3.9
```
Comment (eaidova) [inline .ci/ignore_pip_conflicts.txt:18]: should some warning be provided in notebook for python3.8?

**Review by eaidova — APPROVED** [review]

File: notebooks/285-surya-line-level-text-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Model used for line-level text detection based on [Segformer](https://arxiv.org/pdf/2105.15203.pdf). It has the following features:
+* It is specialized for document OCR. It will likely not work on photos or other images.
+* It is for printed text, not handwriting.
+* The model has trained itself to ignore advertisements.
+* Languages with very different character sets may not work well.
+
+#### Table of contents:
+1. Fetch test image.
+1. Run PyTorch inference.
```
Comment (aleksandr-mokrov) [inline notebooks/285-surya-line-level-text-detection/README.md:20]: Change the nubmering

File: notebooks/285-surya-line-level-text-detection/285-surya-line-level-text-detection.ipynb
Comment (aleksandr-mokrov) [inline notebooks/285-surya-line-level-text-detection/285-surya-line-level-text-detection.ipynb:1]: Fix the table of contents: #run-OpenVINO-model doesn't work

File: notebooks/285-surya-line-level-text-detection/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+![line-level text detection](https://github.com/VikParuchuri/surya/blob/master/static/images/excerpt.png?raw=true)
+
+[**image source*](https://github.com/VikParuchuri/surya)
+
+
+Model used for line-level text detection based on [Segformer](https://arxiv.org/pdf/2105.15203.pdf). It has the following features:
+* It is specialized for document OCR. It will likely not work on photos or other images.
+* It is for printed text, not handwriting.
+* The model has trained itself to ignore advertisements.
+* Languages with very different character sets may not work well.
```
Comment (aleksandr-mokrov) [inline notebooks/285-surya-line-level-text-detection/README.md:16]: Probably it make sense to add next information 

> Text recognition (coming soon)
> Table and chart detection (coming soon)

**Review by aleksandr-mokrov — APPROVED** [review]

**Comment by as-suvorov** [comment]
<p>In order to reuse <code>surya-ocr</code> package functions I implemented wrapper classes for ov model. This classes are fully depend on surya package internals. I fixed surya version to prevent possible inconsistency with surya package updates. </p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1695/discussion/#comment-1936171480'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/285-surya-line-level-text-detection/README.md (hunk 2 above)
Comment (as-suvorov) [reply notebooks/285-surya-line-level-text-detection/README.md:20]: This is markdown feature that numbers don't have to be in numerical order: https://www.markdownguide.org/basic-syntax/#ordered-lists. The markdown application will render lists in numerical order anyway.
Or maybe there is another reason for ordering?

File: notebooks/285-surya-line-level-text-detection/README.md (hunk 3 above)
Comment (as-suvorov) [reply notebooks/285-surya-line-level-text-detection/README.md:16]: I would better not advertise this features as it's unclear when they will be added. It is also unclear if or when we are going to expose them in our notebooks.

**Pull request merged**